test.md

```diff
@@ -0,0 +1,318 @@
+---
+description: Smoketest page
```

Comment:
Does it make sense to add this as a draft, instead of publishing it live?
https://jekyllrb.com/docs/drafts/